app/controllers/trainees/personal_details_controller.rb

```diff
@@ -1,8 +1,40 @@
 module Trainees
   class PersonalDetailsController < ApplicationController
+    before_action :build_nationalities, only: %w[edit update]
+
     def edit
-      @trainee = Trainee.find(params[:id])
+      @personal_details_form = PersonalDetailsForm.new(trainee: trainee)
```

Comment:
When we click on the `Change` link on the confirmation page (see [the prototype](https://register-prototype.herokuapp.com/new-record/contact-details/confirm)) we'll want to use the existing values on the form. As a general pattern would it be worth adding `params: personal_details_params` to the argument here now? Could also optionally kick that can down the road.

Reply:
yeah possibly, although the form object will already be hydrated with the trainee's current attributes if no extra params have been provided but i think i may have to broach this quite soon in one of my other tickets. I'll leave it as it is for now in this PR.